modules/permutiveRtdProvider.js

```diff
@@ -236,11 +239,16 @@ export function getSegments (maxSegs) {
     ac: [..._pcrprs, ..._ppam, ...legacySegs],
     rubicon: readSegments('_prubicons'),
     appnexus: readSegments('_papns'),
-    gam: readSegments('_pdfps')
+    gam: readSegments('_pdfps'),
+    ix: legacySegs
```

Comment:
Currently this only includes standard cohorts (to be transformed to IAB IDs later) but I don't know if this is right.
Maybe it should also include the cohorts from `_ppam` and `_pcrprs` (the same as `ac`)?